docs/content/3.middleware/2.guides/5.caching.md

```diff
@@ -0,0 +1,392 @@
+# Caching
+
+Caching is another powerful technique allowing to boost performance of your VSF middleware application and it's reliability. In the following guide, we will cover subject of caching responses from GET endpoints that are possible to cache.
+
+## Implementation
+
+A caching strategy in middleware has to be implemented per integration. We share a very basic extension showing how to approach it for each of our eCommerce integrations (SAP commerce, commercetools, bigcommerce, Adobe commerce). Copy code of an extension for you eCommerce and assign it to constant inside `middleware.config.js`:
```

Comment:
Adobe commerce?

Reply:
Magento2 is called like that nowadays, Adobe bought them